blockchain/projects_test.go

```diff
@@ -112,105 +111,7 @@ func Benchmark_LoadEmulator(b *testing.B) {
 	})
 }
 
-func Benchmark_GetAccounts(b *testing.B) {
```

Comment:
Why was this removed?

Reply:
Removed it when I removed accounts. Will add back.